tests/agent/test_agent.py

```diff
@@ -295,14 +317,26 @@ def consume(queue: str, on_message: Callable[[EventContext], None]):
     )
 
 
-def test_invalid_config_agent_missing_token(gx_agent_config_missing_token):
-    with pytest.raises(GXAgentConfigError):
-        GXAgent()
+def test_invalid_config_agent_missing_token(set_required_env_vars_missing_token):
+    with pytest.raises(ValidationError):
```

Comment:
error will actually show up sooner as a `GXAgentConfig` validation error.

Reply:
Unsure if fixtures `set_required_env_vars_missing_*`are necessary. May be less SLOC and simpler to read if use the following (pseudocode):
```
@pytest.mark.parametrize("missing_env",["GX_CLOUD_ORGANIZATION_ID","GX_CLOUD_ACCESS_TOKEN"])
test_invalid_config_agent(set_required_env_vars, missing_env):
...
monkeypatch.delenv(missing_env)
...
```